README.md

```diff
@@ -175,13 +172,13 @@ In addition to these, here are some [general log attributes](https://docs.wavefr
 
 # Upgrade
 
-Upgrade the Wavefront Operator (both Collector and Proxy) to a new version by running the following command :
+Upgrade the Observability for Kubernetes Operator (both Kubernetes Metrics Collector and Wavefront proxy) to a new version by running the following command :
```

Comment:
Upgrade the Observability for Kubernetes Operator and underlying agents to a new version